### PR TITLE
empty p-name for notes

### PR DESCRIPTION
**Pull request by tantek**
update docs and examples to suggest empty p-name element for notes and lacking a name, per question in #microformats

### CODE REVIEW

**Review by prtksxna — APPROVED** [review]

File: templates/validate-h-entry.html.php [hunk 1]
```diff
@@ -35,7 +35,9 @@
 				<div class="alert alert-warning">
 					<p>The parsed <code>name</code> name is longer than the content, which is usually a sign is it malformed due to being implicitly rather than explicitly parsed.</p>
 
-					<p>You should always manually specify what the <code>name</code> of a post is. If it doesn’t have one, make the name the same as the content, e.g. <code>&lt;div class=&quot;e-content p-name&quot;>…</code></p>
+					<p>You should always manually specify what the <code>name</code> of a post is. 
+					   If it doesn’t have one, make the name explicitly empty, 
+					   e.g. <code>&lt;span class=&quot;p-name&quot;>&lt;/span>…</code></p>
```
Comment (Lewiscowles1986) [inline templates/validate-h-entry.html.php:40]:
```suggestion
					   e.g. <code>&lt;span class="p-name"&gt;&lt;/span&gt;…</code>
					</p>
```

The tag opening and closing are invalid elements, but the quotes I think are safe, and only get escaped for buggy implementations of the spec AFAIK.

File: templates/validate-h-entry.html.php [hunk 2]
```diff
@@ -210,7 +213,11 @@
 
 	<ul>
 		<li><code>e-content</code> — the main content of the post</li>
-		<li><code>p-name</code> — if your post has a name, use this classname. Otherwise, (if for example the post is a <a href="https://indieweb.org/note">note</a>), either leave it off or apply to the same element as <code>e-</code>.</li>
+		<li><code>p-name</code> — if your post has a name, use this classname. Otherwise, 
+			(if for example the post is a <a href="https://indieweb.org/note">note</a>), 
+			make an explicitly empty name like: 
+			&lt;span class=&quot;p-name&quot;>&lt;/span>
```
Comment (Lewiscowles1986) [inline templates/validate-h-entry.html.php:219]:
```suggestion
			&lt;span class=&quot;p-name&quot;&gt;&lt;/span&gt;
```

File: templates/validate-h-entry.html.php [hunk 3]
```diff
@@ -192,10 +194,11 @@
 		<?php else: ?>
 			<h3>No h-entry found</h3>
 
-			<p>Adding h-entry markup to posts on your site allows computers to understand them as easily as humans can, without publishing separate copies. All you need to do is add microformats2 h-entry classnames, for example:</p>
+			<p>Adding h-entry markup to posts on your site allows computers to understand them as easily as humans can, 
+			   without publishing separate copies. All you need to do is add microformats2 h-entry classnames, for example:</p>
 
-			<pre><code>&lt;article class=&quot;h-entry&quot;>
-  &lt;div class=&quot;e-content p-name&quot;>Hello world! This is my first indieweb post.&lt;/div>
+			<pre><code>&lt;article class=&quot;h-entry&quot;>&lt;span class=&quot;p-name&quot;>&lt;/span>
```
Comment (Lewiscowles1986) [inline templates/validate-h-entry.html.php:200]:
```suggestion
			<pre><code>&lt;article class="h-entry"&gt;&lt;span class=&quot;p-name&quot;&gt;&lt;/span&gt;
```

**Review by Lewiscowles1986 — APPROVED** [review]
I'm not sure why some characters are escaped and others are not, but it certainly communicates clearly. I've issued suggestions, but I don't think they are blocking because they are unbalanced, so don't form a tag.

**Comment by gRegorLove** [comment]
I think all the in-production parsers handle the spec update that reduces instances of implied p-name, so I'm not sure this is a needed recommendation. [chat log](https://chat.indieweb.org/microformats/2020-04-16#t1587068249228600)

**Pull request closed without merging**